rotkehlchen/chain/evm/decoding/hop/constants.py

```diff
@@ -0,0 +1,21 @@
+from dataclasses import dataclass
+from typing import Final
+
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.chain.evm.decoding.types import CounterpartyDetails
+from rotkehlchen.types import ChecksumEvmAddress
+
+
+@dataclass
+class HopBridge:
+    address: ChecksumEvmAddress
+    asset: Asset
+    amm_wrapper: ChecksumEvmAddress | None = None
```

Comment:
this is not the right place to have this data structure. Crete a file types or similar. Also add a docstring explaining what are the different fields and what this is used for